app/views/find/search/locations/_form.html.erb

```diff
@@ -96,31 +96,20 @@
             <div class="govuk-radios__conditional <%= "govuk-radios__conditional--hidden" unless params[:l] == "3" %>"
               id="query-container" data-qa="by-provider-conditional">
               <div class="govuk-form-group <%= "govuk-form-group--error" if provider_error? %>" data-module="track-no-provider-results">
-                <!-- if cached providers are present render autocomplete else render sans-js provider search -->
-                <% if select_provider_options(@providers) %>
-                  <% if provider_error? %>
-                     <p class="govuk-error-message" id="provider-error" data-qa="provider-error">
-                     <span class="govuk-visually-hidden">Error: </span><%= flash[:error].last %></p>
-                  <% end %>
-                  <% aria_described_by = "provider-hint" %>
-                  <% aria_described_by += " provider-error" if provider_error? %>
-                   <%= form.govuk_collection_select "provider.provider_name", select_provider_options(@providers), :id, :name, label: { text: "School, university or other training provider" }, hint: { text: "Enter the name or provider code" } %>
-                <% else %>
-                  <%= form.label "provider.provider_name", { class: "govuk-label", for: "provider" } do %>
-                    School, university or other training provider
-                    <div class="govuk-hint">Enter the name or provider code</div>
-                    <% if provider_error? %>
-                    <p class="govuk-error-message" id="provider-error" data-qa="provider-error">
-                    <span class="govuk-visually-hidden">Error: </span><%= flash[:error].last %>
-                  </p>
-                    <% end %>
-                  <% end %>
-                  <%= form.text_field "provider.provider_name",
-                    id: "provider",
-                    value: params["provider.provider_name"],
-                    class: "govuk-input",
-                    data: { qa: "provider-search" } %>
+                <% if provider_error? %>
+                   <p class="govuk-error-message" id="provider-error" data-qa="provider-error">
+                   <span class="govuk-visually-hidden">Error: </span><%= flash[:error].last %></p>
                 <% end %>
+                <% aria_described_by = "provider-hint" %>
+                <% aria_described_by += " provider-error" if provider_error? %>
+                <%= render Find::AutocompleteComponent.new(
+                                                form_field: form.govuk_select(
```

Comment:
Can we sort out the indentation here?

Reply:
Done: https://github.com/DFE-Digital/publish-teacher-training/pull/3392/commits/b4df43822c0850d27ce09727d0bbf7e2788f7f22

Does that ERB linter actually help us 😠